dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowMode.java

```diff
@@ -40,5 +40,9 @@
     /**
      * A degenerate workflow
      */
-    SERVICE
+    SERVICE,
+    /**
+     * A workflow that uses a .dockstore.yml
+     */
+    DOCKSTORE_YML
```

Comment:
I wonder if the name should have something to indicate workflow in it? What if we add support for tools and dockstore.yml?

Reply:
I think eventually we would just want one version of .dockstore.yml that works across all entry types.